supertokens_python/recipe/thirdpartypasswordless/recipe.py

```diff
@@ -165,6 +165,12 @@ async def is_email_verified(user_id: str, email: str, user_context: Dict[str, An
         else:
             self.email_delivery_ingredient = email_delivery_ingredient
 
+        email_delivery_ingredient = ingredients.email_delivery if ingredients else None
+        if email_delivery_ingredient is None:
+            self.email_delivery_ingredient = EmailDeliveryIngredient(self.config.get_email_delivery_config(recipe_implementation))
+        else:
+            self.email_delivery_ingredient = email_delivery_ingredient
```

Comment:
This is already happening right above. Why duplicate?

Reply:
Removed.

Reply:
@rishabhpoddar Please resolve this.